governance/votes/Executive vote - July 2, 2021.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Template - [Executive Vote] Rates Changes, July Core Unit Budgets, and PSM DC-IAM - July 2, 2021
+summary: Vote for Rates Changes, July Core Unit Budgets, and PSM DC-IAM
+date: 2021-07-02T00:00:00.000Z
```

Comment:
Should this be 16:00 when they are scheduled to go up?

Reply:
Mm, we never hit that anyway. I would rather keep it at midnight, I think.